CheatSheets/vim editor_cheatsheet.md

```diff
@@ -0,0 +1,142 @@
+# Introduction
+
+Vim is a widely used, open-source Unix text editor. Learning to use Vim commands is a matter of practice and experience. That is why it is handy to have a helpful reference sheet while mastering them.
+
+## How to open a file in vim
+```vim
+$ vim <file path>
+```
+Example:
+```vim
+$ vim './myRepo/README.md'
+or
+$ vim README.md 
+```

```

Comment:
```suggestion
| Commands | Descriptions | Examples |
| ------------- | ------------- | ------------ |
| `$ vim <file path>` | Open a file in vim | - `$ vim './myRepo/README.md'` <br /> - `$ vim README.md ` |

```
## Basic Commands

| Commands | Descriptions | Examples |
| ------------- | ------------- | ------------ |
| `$ vim <file path>` | Open a file in vim | - `$ vim './myRepo/README.md'` <br /> - `$ vim README.md ` |

Please, give them a `Tabular format` rather than a `block code`. It will be better to `visualize`, `understand`, and be `easy to read`.